staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/conversion.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I runned `make clean` before `hack/update-all.sh` and it succeed.
Now this file is moved to first commit.
Thanks for your concern.